common/djangoapps/third_party_auth/models.py

```diff
@@ -468,6 +487,10 @@ def get_config(self):
         idp_class = get_saml_idp_class(self.identity_provider_type)
         return idp_class(self.idp_slug, **conf)
 
+    @property
+    def required_extra_attributes(self):
```

Comment:
Again, not a very good rationale -- I believe at first I was trying some weird solution that required me to use it this way, but I reworked it and no longer needed it but kept it as a property. I'm going to revert it to a class variable.